examples/Atmos/heldsuarez.jl

```diff
@@ -11,13 +11,9 @@ using CLIMA.VTK: writevtk, writepvtu
 using CLIMA.GenericCallbacks: EveryXWallTimeSeconds, EveryXSimulationSteps
 using CLIMA.MPIStateArrays: euclidean_distance
 using CLIMA.PlanetParameters: R_d, grav, MSLP, planet_radius, cp_d, cv_d, day
-using CLIMA.MoistThermodynamics: air_density, total_energy, soundspeed_air, internal_energy, air_temperature
-using CLIMA.Atmos: AtmosModel, SphericalOrientation, NoReferenceState,
-                   DryModel, NoPrecipitation, NoRadiation, NoSubsidence, NoFluxBC,
-                   ConstantViscosityWithDivergence,
-                   vars_state, vars_aux,
-                   Gravity, Coriolis,
-                   HydrostaticState, IsothermalProfile
```

Comment:
Should I also be checking if we might need to use any of the names that were removed from the specific `using` calls?

Reply:
Such were potentially not being used, so it would be better to check what is being exported across the board.